static/js/publisher/market/stickyListingBar.js

```diff
@@ -0,0 +1,20 @@
+export default function() {
+  const stickyBar = document.querySelector(".snapcraft-p-sticky");
+  const listings = document.querySelector(
+    ".u-float--left.p-heading--three.u-no-margin--bottom"
```

Comment:
This is overly specific selector that can easily change if we change something in HTML.
If we want to identify listings element we should add an `id` to it or some `js-listings` class that is specific for this use case.

Reply:
This has been removed completely